etc/plf/setenv-customize.sh

```diff
@@ -64,7 +64,7 @@ fi
 
 # eXo Onlyoffice addon
 if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
-    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=${DEPLOYMENT_APACHE_VHOST_ALIAS}:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```

Comment:
What will happen when there will be no vhost declared ?
Why not using DEPLOYMENT_URL ?  https://github.com/exoplatform/adt/blob/master/_functions.sh#L1144

Reply:
ok it's the url not the host but IMO you should introduce something similar

Reply:
I totally agree

Reply:
What do you think of introducing a new var : 
env_var  "ONLYOFFICE_EXT_HOST_PORT" $(echo ${DEPLOYMENT_URL} | sed 's/https\?:\/\///')

?